src/main/scala-2/com/evolutiongaming/random/Random.scala

```diff
@@ -36,62 +42,93 @@ object Random {
     }
   }
 
-  type SeedT[A] = cats.data.StateT[Id, Seed, A]
+  /** The random number generator for a single specific type `A`.
+    *
+    * It takes some `state1` as an input and returns a new `state2` and a random
+    * value of type `A`.
+    *
+    * Technically, it is just a function from `(Seed)` to `(Seed, A)`.
+    *
+    * `StateT` is used instead of a plain function, got have the ability to
```

Comment:
I am not sure, if `got` is a correct word there:
```suggestion
    * `StateT` is used instead of a plain function, it has the ability to
```